### ISSUE TITLE
undefined method `content_tag' 

### ISSUE THREAD

**Issue by Amnesthesia** [Apipie/apipie-rails#354]
No matter what I do, it seems that as soon as I click to view the details of any resource, I get this error.

undefined method `content_tag' for #<Apipie::ApipiesController:0x007fecd8cd6ff0>

I'm on Ruby 2.1.4 and Rails 4.2.1

Extracted source (around line #4):
```ruby
  def heading(title, level=1)
    content_tag("h#{level}") do
      title
    end
  end
```

**Top stack trace:**

1. apipie-rails (0.3.4) app/helpers/apipie_helper.rb:4:in `heading'
2. actionpack (4.2.1) lib/abstract_controller/helpers.rb:67:in `heading'
3. apipie-rails (0.3.4) app/views/apipie/apipies/_method_detail.erb:4:in `__home_amnesthesia__rvm_gems_ruby________ow_n_gems_apipie_rails_______app_views_apipie_4. apipies__method_detail_erb___4240183086102969573_60079080'
5. actionview (4.2.1) lib/action_view/template.rb:145:in `block in render'
6. activesupport (4.2.1) lib/active_support/notifications.rb:166:in `instrument'
7. actionview (4.2.1) lib/action_view/template.rb:333:in `instrument'
8. actionview (4.2.1) lib/action_view/template.rb:143:in `render'
9. actionview (4.2.1) lib/action_view/renderer/partial_renderer.rb:339:in `render_partial'

**Comment by saneshark** [comment]
@Amnesthesia you are not alone: see: #350

You might be ok if you drop down to 0.3.3 in the interim.

**Comment by Seralto** [comment]
Same problem.

**Comment by waleedasif322** [comment]
Try putting the latest version in your Gemfile using github, the problem was fixed for me like this:
`gem 'apipie-rails', :github => 'Apipie/apipie-rails'`

**Comment by Seralto** [comment]
Great waleedasif322, problem solved ;)

**Comment by FPierre** [comment]
Same problem, same solution (`gem 'apipie-rails', :github => 'Apipie/apipie-rails'`)

**Comment by IlyaBylich** [comment]
@FPierre It's better to specify revision that has this fix and nothing more. You don't want to get unexpected changes after running `bundle udpate`, right?
``` ruby
gem 'apipie-rails', github: 'Apipie/apipie-rails', ref: '928bd858fd14ec67eeb9483ba0d43b3be8339608'
```

**Comment by jasonswett** [comment]
I was having the same problem on Rails 3.2 and was able to get it working with commit `a74f196f99fb33612c3e4982ade3077ab663a510`.

**Comment by brunojppb** [comment]
@IlyaBylich +1

**Comment by jayelkaake** [comment]
+1 - Thanks guys!

**Comment by wesf90** [comment]
I hate to +1 again, but this was an excellent solution to this tricky problem. Thanks @iliabylich

**Comment by lucasecf** [comment]
@IlyaBylich +1

**Comment by Thomsen** [comment]
@iliabylich +1 Thanks!

gem uninstall apipie-rails
bundle install

**Comment by ChristianHoj** [comment]
Appears to be fixed with PR #350 and included in release 0.3.5 on October 21
